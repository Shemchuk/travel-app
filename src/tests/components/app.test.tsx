import React from 'react';
import renderer from 'react-test-renderer';

import App from '../../components/app';

describe('App', () => {
  test('renders correctly', () => {
    const app = renderer
      .create(<App />)
      .toJSON();
    expect(app).toMatchSnapshot();
  });
});
