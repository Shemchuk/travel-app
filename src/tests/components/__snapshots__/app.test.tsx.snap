// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
Array [
  <header
    className="header"
  >
    <div
      className="header_wrapper"
    >
      <div
        className="header_logo"
      >
        <a
          href="#/"
          onClick={[Function]}
        >
          travel the world
        </a>
      </div>
      <div
        className="header_menu"
      >
        <nav
          className="header_nav"
        >
          <ul>
            <li>
              <a
                href="#/"
                onClick={[Function]}
              >
                Home
              </a>
            </li>
            <li>
              <a
                href="#/about"
                onClick={[Function]}
              >
                about
              </a>
            </li>
            <li>
              <a
                href="#/login"
                onClick={[Function]}
              >
                login
              </a>
            </li>
            <li>
              <select
                className="lang"
                id="lang"
                name="lang"
              >
                <option
                  value="EN"
                >
                  EN
                </option>
                <option
                  value="RU"
                >
                  RU
                </option>
                <option
                  value="BY"
                >
                  BY
                </option>
              </select>
            </li>
          </ul>
        </nav>
        <div
          className="header_search"
        >
          <img
            alt="search"
            src="test-file-stub"
          />
          <input
            type="search"
          />
        </div>
      </div>
    </div>
  </header>,
  <div
    className="countries-wrapper"
  >
    <a
      href="#/country/1"
      onClick={[Function]}
    >
      <div
        className="country-card"
      >
        <span
          className="title_cart"
        >
          Franse
        </span>
        <span
          className="sub-title_cart"
        >
          Paris
        </span>
        <span
          className="more"
        >
          start your journey...
        </span>
      </div>
    </a>
    <a
      href="#/country/1"
      onClick={[Function]}
    >
      <div
        className="country-card"
      >
        <span
          className="title_cart"
        >
          Franse
        </span>
        <span
          className="sub-title_cart"
        >
          Paris
        </span>
        <span
          className="more"
        >
          start your journey...
        </span>
      </div>
    </a>
    <a
      href="#/country/1"
      onClick={[Function]}
    >
      <div
        className="country-card"
      >
        <span
          className="title_cart"
        >
          Franse
        </span>
        <span
          className="sub-title_cart"
        >
          Paris
        </span>
        <span
          className="more"
        >
          start your journey...
        </span>
      </div>
    </a>
    <a
      href="#/country/1"
      onClick={[Function]}
    >
      <div
        className="country-card"
      >
        <span
          className="title_cart"
        >
          Franse
        </span>
        <span
          className="sub-title_cart"
        >
          Paris
        </span>
        <span
          className="more"
        >
          start your journey...
        </span>
      </div>
    </a>
    <a
      href="#/country/1"
      onClick={[Function]}
    >
      <div
        className="country-card"
      >
        <span
          className="title_cart"
        >
          Franse
        </span>
        <span
          className="sub-title_cart"
        >
          Paris
        </span>
        <span
          className="more"
        >
          start your journey...
        </span>
      </div>
    </a>
    <a
      href="#/country/1"
      onClick={[Function]}
    >
      <div
        className="country-card"
      >
        <span
          className="title_cart"
        >
          Franse
        </span>
        <span
          className="sub-title_cart"
        >
          Paris
        </span>
        <span
          className="more"
        >
          start your journey...
        </span>
      </div>
    </a>
    <a
      href="#/country/1"
      onClick={[Function]}
    >
      <div
        className="country-card"
      >
        <span
          className="title_cart"
        >
          Franse
        </span>
        <span
          className="sub-title_cart"
        >
          Paris
        </span>
        <span
          className="more"
        >
          start your journey...
        </span>
      </div>
    </a>
    <a
      href="#/country/1"
      onClick={[Function]}
    >
      <div
        className="country-card"
      >
        <span
          className="title_cart"
        >
          Franse
        </span>
        <span
          className="sub-title_cart"
        >
          Paris
        </span>
        <span
          className="more"
        >
          start your journey...
        </span>
      </div>
    </a>
    <a
      href="#/country/1"
      onClick={[Function]}
    >
      <div
        className="country-card"
      >
        <span
          className="title_cart"
        >
          Franse
        </span>
        <span
          className="sub-title_cart"
        >
          Paris
        </span>
        <span
          className="more"
        >
          start your journey...
        </span>
      </div>
    </a>
    <a
      href="#/country/1"
      onClick={[Function]}
    >
      <div
        className="country-card"
      >
        <span
          className="title_cart"
        >
          Franse
        </span>
        <span
          className="sub-title_cart"
        >
          Paris
        </span>
        <span
          className="more"
        >
          start your journey...
        </span>
      </div>
    </a>
    <a
      href="#/country/1"
      onClick={[Function]}
    >
      <div
        className="country-card"
      >
        <span
          className="title_cart"
        >
          Franse
        </span>
        <span
          className="sub-title_cart"
        >
          Paris
        </span>
        <span
          className="more"
        >
          start your journey...
        </span>
      </div>
    </a>
    <a
      href="#/country/1"
      onClick={[Function]}
    >
      <div
        className="country-card"
      >
        <span
          className="title_cart"
        >
          Franse
        </span>
        <span
          className="sub-title_cart"
        >
          Paris
        </span>
        <span
          className="more"
        >
          start your journey...
        </span>
      </div>
    </a>
    <a
      href="#/country/1"
      onClick={[Function]}
    >
      <div
        className="country-card"
      >
        <span
          className="title_cart"
        >
          Franse
        </span>
        <span
          className="sub-title_cart"
        >
          Paris
        </span>
        <span
          className="more"
        >
          start your journey...
        </span>
      </div>
    </a>
    <a
      href="#/country/1"
      onClick={[Function]}
    >
      <div
        className="country-card"
      >
        <span
          className="title_cart"
        >
          Franse
        </span>
        <span
          className="sub-title_cart"
        >
          Paris
        </span>
        <span
          className="more"
        >
          start your journey...
        </span>
      </div>
    </a>
  </div>,
]
`;
